### ISSUE TITLE
Back to the future :)

### ISSUE THREAD

**Issue by JanHBade** [DellAquila/GameTime#1]
![20210618140151_1](https://user-images.githubusercontent.com/3524802/122558303-3921d380-d03e-11eb-967a-f17319d0d72e.jpg)

14:01 is the right one...

**Comment by JanHBade** [comment]
![20210624210758_1](https://user-images.githubusercontent.com/3524802/123319184-6388fa80-d530-11eb-90b0-dcd1c9391d43.jpg)

**Comment by DellAquila** [comment]
Thats super strange... Happening with some players... :( You are in Europe? Seems like German language causes it...

**Comment by JanHBade** [comment]
yes German is selected in the game (and also my native language)

but the numbers are the same in English and German ;)

**Comment by leasoncre** [comment]
just found this issue myself (time is 00:03, not 00:30, see hud time)
English, USA, EST
![20211228000325_1](https://user-images.githubusercontent.com/13475541/147530343-3a9f92ba-aaed-4b01-a049-ff760dbbaadd.jpg)

**Comment by leasoncre** [comment]
> just found this issue myself (time is 00:03, not 00:30, see hud time) English, USA, EST 

10 min later and it's fixed itself. must be some loose memory/code. as i watched it tick from 02-to-03 minutes, but display it in the 10's digit

EDIT: single digit emulation justification issue?
